Strategy and Structure.md

```diff
@@ -0,0 +1,36 @@
+# Strategy and Structure
+
+## What
+
+CLI Unification is an evolution of Salesforce’s CLI strategy. We are unifying all the CLIs across Salesforce’s many brands as our first step in providing a unified cross-cloud developer experience. Our ultimate goal is to provide one set of tools for all Salesforce developers to write and deploy their Salesforce Apps. 
+
+We're evolving the taxonomy of our Salesforce CLI to bring together existing capabilities from Salesforce CLI and other CLIs, such as the Heroku CLI, MuleSoft CLI, etc., to deliver net new, cross-cloud commands under the new executable, `sf`. The new executable will come bundled with `sfdx` in the same installer and work in conjunction with `sfdx` commands, until sf reaches full feature parity with sfdx.
+
+## Structure
+
+Most of the CLI commands in sf are designed around the actions you take or the job you perform when running a command, instead of focusing on product or brand names. For this universal set of jobs to be done (JTBD), the topics are cross cloud and give you the ability to perform the same type of action for multiple Salesforce products. Those actions include the ability to *deploy* your code/metadata, *generate* scaffolded files for different types of resources, and connect to your *environments*. These topics are opinionated, and structured so that they are, by default, cross cloud. 
```

Comment:
```suggestion
Most CLI commands in `sf` are designed around the actions you take or the job you perform when running a command, instead of focusing on product or brand names. For this universal set of jobs to be done (JTBD), the topics are cross cloud and give you the ability to perform the same type of action for multiple Salesforce products. Those actions include the ability to *deploy* your code/metadata, *generate* scaffolded files for different types of resources, and connect to your *environments*. These topics are opinionated, and structured so that they are, by default, cross cloud. 
```